src/region/RegionContainer.tsx

```diff
@@ -36,11 +37,16 @@ export type Props = {
 
 export const mapStateToProps = (state: AppState, { location }: { location: number }): Props => {
     const staged = getCreatorStagedForLocation(state, location);
+    const isDiscoverabilityFeatureEnabled = getIsDiscoverabilityFeatureEnabled(state);
+
+    const isCreating = isDiscoverabilityFeatureEnabled
```

Comment:
This seems to defeat the purpose of `Mode` somewhat. Are we concerned this will cause issues with other areas that rely on mode being "None"? I don't think it will add the new `ba-is-create--region` class by default, either, right?

Reply:
It seems like the Preview SDK should set the Annotations SDK mode to Mode.REGION on load, instead of handling two modes here.

Reply:
This makes sense. I can try addressing that in the separate FF change.